modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp

```diff
@@ -77,7 +77,7 @@ entriesChecker.setRememberCheckBoxStateURLRegex("^(?!.*" + liferayPortletRespons
 							latestFileVersion = fileEntry.getLatestFileVersion();
 						}
 
-						if ((dlSearchContainer.getRowChecker() == null) && (DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DELETE) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.UPDATE))) {
+						if ((dlSearchContainer.getRowChecker() == null) && (DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DELETE) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DOWNLOAD) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.UPDATE))) {
```

Comment:
this should be done on
line https://github.com/liferay/liferay-portal/blob/72447fc3708065aa8e158956431ecc036bc63139/modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp#L395 too?

Reply:
Folders do not have the dowload permission

Reply:
Hey @AliciaGarciaGarcia 

I hadn't noticed that this permission was added to Folders lately.

I'll add it.